src/actions/actionTypes.js

```diff
@@ -7,10 +7,12 @@ export const SIGN_IN_SUCCESS = 'SIGN_IN_SUCCESS';
 export const CREATE_TARGET = 'CREATE_TARGET';
 export const LOAD_TARGETS = 'LOAD_TARGETS';
 export const UPDATE_TARGET = 'UPDATE_TARGET';
-export const UPDATE_FREE_TARGET = 'UPDATE_FREE_TARGET';
 export const DELETE_TARGET = 'DELETE_TARGET';
 export const RESET_TARGETS = 'RESET_TARGETS';
+export const UPDATE_FREE_TARGET = 'UPDATE_FREE_TARGET';
 export const RESET_FREE_TARGET = 'RESET_FREE_TARGET';
+export const UPDATE_SELECTED_TARGET = 'UPDATE_SELECTED_TARGET';
```

Comment:
You are repeating too much code. You can use this library 'mirror-creator'.
So you simply do: 
`export const mirrorCreator(['DELETE_TARGET, 'CREATE_TARGET', ....]), 
`which is equal to have 
```
{
    DELETE_TARGET = 'DELETE_TARGET',
    CREATE_TARGET = 'CREATE_TARGET'
}
```